include/odp/api/spec/shared_memory.h

```diff
@@ -44,6 +44,14 @@ extern "C" {
 
 /**
  * Application SW only, no HW access
+ *
+ * When set, application will not share the reserved memory with HW
+ * accelerators. However, leaving this flag to zero does not guarantee that
+ * the reserved memory can be accessed from HW, and thus usage of this flag is
+ * considered deprecated.
```

Comment:
Perhaps use @deprecated tag here. It's easier to notice especially in the doxygen documentation (e.g. [ODP_PKTIO_PARSER_LAYER_NONE](https://opendataplane.github.io/odp/group__odp__packet__io.html#ga8cb8ad8f5ccdf9f29783b39c54815984)).

Reply:
Fixed in v2